std_msgs/Header header                  # time stamp
int64   id
float64 completion
float64 press_rate
int64   press_counter